server/heroku/logs.go

```diff
@@ -20,13 +21,14 @@ type PostLogsForm struct {
 func (h *PostLogs) ServeHTTPContext(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
 	a, err := findApp(ctx, h)
 	if err != nil {
-		return err
+		return fmt.Errorf("error finding app: %v", err)
```

Comment:
Thats what confuses me about the current error handling, some have been annotated and it's "safe" to just return `err` while others are `EOF` and suck.

Maybe it's worth biting the bullet and getting after: https://github.com/remind101/empire/issues/812 with a clear standard on how to handle errors?